### ISSUE TITLE
I am not able to use the bootstrap in it. All the div are printing in same row when i am using inside loop

### ISSUE THREAD

**Issue by bjpatel2001** [barryvdh/laravel-dompdf#516]
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Bootstrap Example</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  <style>
    .invoice-box {
      max-width: 800px;

      margin: auto;
      padding: 30px;
      border: 1px solid #eee;
      box-shadow: 0 0 10px rgba(0, 0, 0, .15);
      font-size: 16px;
      line-height: 24px;
      color: #555;
      font-family: 'Lato', sans-serif;
    }
    .lead {
      margin-bottom: 15px;
      float: left;

    }
    .span {
      font-weight: bold;
    }
  </style>
</head>
<body>

<div class="container invoice-box">

  <div class="row">
    <div class="col-md-12">

      @foreach($familyData as $data)
      <div class="col-md-6 lead">
        <div class="row">
          <div class="col-md-12">
            <span class="span">Name:</span> {{$data->name .''.$data->surname}}
          </div>
          <div class="col-md-12">
            <span class="span">Address:</span> {{$data->address or ''}}
          </div>
          <div class="col-md-12">
            <span class="span">City:</span> {{$data->city_name or ''}}
          </div>
          <div class="col-md-12">
            <span class="span">Taluka:</span> {{$data->taluka or ''}}
          </div>
          <div class="col-md-12">
            <span class="span">District:</span> {{$data->district or ''}}
          </div>
          <div class="col-md-12">
            <span class="span">State:</span> {{$data->state_name or ''}}
          </div>
          <div class="col-md-12">
            <span class="span">Pincode:</span> {{$data->pincode or ''}}
          </div>
        </div>
      </div>
      @endforeach

    </div>
  </div>

</div>

</body>
</html>

**Comment by kapitannwel** [comment]
shouldn't you use bootstrap.min.css?

**Comment by bjpatel2001** [comment]
It is there I have used it but still it is printing everything in one row with overlapping. Attached is the sample of which the pdf is generated
[17 (1).pdf](https://github.com/barryvdh/laravel-dompdf/files/2360275/17.1.pdf)

**Comment by kapitannwel** [comment]
i still think i am seeing dot js or am i just missing it?

**Comment by bjpatel2001** [comment]
It is there but some how it is not showing here above. But i tried using it. Still not luck. I think col-md are not working. Everything is been printed in same raw. Am i missing some thing in the html code i  am un able to figure it out.

**Comment by Sidney-Dev** [comment]
You need to use inline styling if am correct. No external stylesheets
